src/pages/Login/index.tsx

```diff
@@ -1,83 +1,22 @@
-  const dispatch = useDispatch<AppDispatch>()
 
-  const handleSubmit = async (e: any) => {
-    e.preventDefault()
-    try {
-      await dispatch(login({ email, password }))
-      if (localStorage.getItem('accessToken')) {
-        if (rSelected === 1) {
-          navigate('/customer')
-        } else navigate('/admin')
-      }
-    } catch (error: any) {
-      console.log(error?.message)
-    }
-  }
-  return (
-    <div className="w-50 mt-20 container">
-      <div className="d-flex flex-col justify-content-center align-items-center">
-        <h1>Login</h1>
-        <ButtonGroup className="my-5">
-          <Button
-            color="primary"
-            outline
-            onClick={() => setRSelected(1)}
-            active={rSelected === 1}
-          >
-            Customer
-          </Button>
-          <Button
-            color="primary"
-            outline
-            onClick={() => setRSelected(2)}
-            active={rSelected === 2}
-          >
-            Admin
-          </Button>
-        </ButtonGroup>
-        <Form className="text-center w-100" onSubmit={(e) => handleSubmit(e)}>
-          <FormGroup className="d-flex justify-content-center align-items-center">
-            <Label for="exampleEmail" className="w-25 text-left">
-              Email:
-            </Label>
-            <Input
-              id="exampleEmail"
-              name="email"
-              type="email"
-              className="w-50 ml-3"
-              value={email}
-              onChange={(e) => setEmail(e.target.value)}
-            />
-          </FormGroup>
-          <FormGroup className="d-flex justify-content-center align-items-center">
-            <Label for="examplePassword" className="w-25 text-left">
-              Password:
-            </Label>
-            <Input
-              id="examplePassword"
-              name="password"
-              type="password"
-              className="w-50 ml-3"
-              value={password}
-              onChange={(e) => setPassword(e.target.value)}
-            />
-          </FormGroup>
-          <Button className='w-50' color="primary">Submit</Button>
-        </Form>
-      </div>
-    </div>
-  )
+const Login = () => {
+    return(<div>
+        <Decorator></Decorator>
+        <Header title= "Login" className="text-4xl mt-40"></Header>
+        <Input id="E-mail" type="text" className="mt-8" ></Input>
+        <Input id="Password" type="password" className="mt-8"></Input>
+        <AlreadyHaveAccount color="text-primary" className="mt-8 justify-center" text="Forgot password?"></AlreadyHaveAccount>
+        <ConfirmButton title="LOG IN" className="mt-8"></ConfirmButton>
+        <AlreadyHaveAccount text="Don't have an account? " color="text-grayAccount" className="mt-8 justify-center" option={{color:"text-primary", content:"Sign Up"}}></AlreadyHaveAccount>
+        <SignInWith color={{text: "grayAccount", divider:"bg-[#B3B3B3] bg-opacity-50"}} className="mt-12"></SignInWith>
```

Comment:
`#B3B3B3` does this value exist in tailwind.config file ? If then, use the value there instead